File: 프로그래머스/Lv.0/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        int answer = 0;
        
        if(Integer.compare(a, b) == 0 && Integer.compare(b, c) == 0){
            answer = (a + b + c) * (int)(Math.pow(a, 2) + Math.pow(b, 2) + Math.pow(c, 2)) * (int)(Math.pow(a, 3) + Math.pow(b, 3) + Math.pow(c, 3));
        }else if(Integer.compare(a, b) == 0 || Integer.compare(b, c) == 0 || Integer.compare(a, c) == 0){
            answer = (a + b + c) * (int)(Math.pow(a, 2) + Math.pow(b, 2) + Math.pow(c, 2));
        }else{
            answer = a + b + c;
        }
        
        
        return answer;
    }
}

File: 프로그래머스/Lv.0/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    answer = 0
    count = len(''.join(set(f"{a}{b}{c}")))
    if count == 3:
        answer = a + b + c
    elif count == 2:
        answer = (a+b+c) * (a**2 + b**2 + pow(c,2))
    else:
        answer = (a+b+c) * (a**2 + b**2 + pow(c,2)) * (a**3 + b**3 + pow(c,3))
    return answer